simtools/simtel/simtel_light_emission.py

```diff
@@ -0,0 +1,552 @@
+            angle_phi = 180 - angles[1]
+            command += super()._config_option("telescope_theta", f"{angle_theta}")
+            command += super()._config_option("telescope_phi", f"{angle_phi}")
+        else:
+            command += super()._config_option("telescope_theta", 0)
+            command += super()._config_option("telescope_phi", 0)
+
+        command += super()._config_option("power_law", "2.68")
+        command += super()._config_option(
+            "input_file", f"{self.output_directory}/{self.le_application[0]}.iact.gz"
+        )
+        command += super()._config_option(
+            "output_file",
+            f"{self.output_directory}/"
+            f"{self.le_application[0]}_{self.le_application[1]}.simtel.gz\n",
+        )
+
+        return command
+
+    def _create_postscript(self, **kwargs):  # pylint: disable=unused-argument
+        """
+        writes out post-script file using read_cta
+        """
+        postscript_dir = self.output_directory.joinpath("postscripts")
+        postscript_dir.mkdir(parents=True, exist_ok=True)
+
+        command = str(self._simtel_source_path.joinpath("hessioxxx/bin/read_cta"))
+        command += " --min-tel 1 --min-trg-tel 1"
+        command += " -q --integration-scheme 4 --integration-window 7,3 -r 5"
+        command += " --plot-with-sum-only"
+        command += " --plot-with-pixel-amp --plot-with-pixel-id"
+        # command += f" --plot-with-title 'tel {self._telescope_model.name}"
+        command += (
+            f" -p {postscript_dir}/"
+            f"{self.le_application[0]}_{self.le_application[1]}_"
+            f"d_{int(self.distance.to(u.m).value)}.ps"
+        )
+        command += (
+            f" {self.output_directory}/"
+            f"{self.le_application[0]}_{self.le_application[1]}.simtel.gz\n"
+        )
+        # ps2pdf required, now only store postscripts
+        # command += f"ps2pdf {self.output_directory}/{self.le_application}.ps
+        #  {self.output_directory}/{self.le_application}.pdf"
+        return command
+
+    def plot_simtel_ctapipe(self, return_cleaned=0):
+        """
+        reads in simtel file and plots reconstructed photo electrons via ctapipe
+        """
+        filename = (
+            f"{self.output_directory}/"
+            f"{self.le_application[0]}_{self.le_application[1]}.simtel.gz"
+        )
+        source = EventSource(filename, max_events=1)
+        event = None
+        for event in source:
+            print(event.index.event_id)
+        tel_id = sorted(event.r1.tel.keys())[0]
+
+        calib = CameraCalibrator(subarray=source.subarray)
+
+        calib(event)
+        geometry = source.subarray.tel[1].camera.geometry
+
+        image = event.dl1.tel[tel_id].image
+        cleaned = image.copy()
+
+        if return_cleaned:
+            mask = tailcuts_clean(
+                geometry, image, picture_thresh=7, boundary_thresh=5, min_number_picture_neighbors=0
```

Comment:
Hardwired cleaning parameters - this is / should be a configuration parameter?

Reply:
Added the command line parameters.